Update issue template (#1168)
mongodb/terraform-provider-mongodbatlas: .github/ISSUE_TEMPLATE/Bug_Report.md

Thank you for opening an issue.  Please note that we try to keep the Terraform MongoDB Atlas Provider issue tracker reserved for bug reports.  Please ensure you check open and closed issues first to ensure your issue hasn't already been reported (if it has been reported add a reaction, i.e. +1, to the issue).  

If your issue relates to Terraform itself, please open it in the Terraform repository https://github.com/hashicorp/terraform/issues.

> Do you have a MongoDB Atlas Support contract?  

Reviewer: ```suggestion
Do you have an active MongoDB Atlas Support contract?  
```

> If Yes: Then a support ticket is the best way for us to assist you with the provider.  Repo issues are for community reporting/assistance only and are best effort. 

Reviewer 2: [q] Could we add a link here?
```suggestion
If Yes: Then a [support ticket](https://support.mongodb.com/) is the best way for us to assist you with the provider.  Repo issues are for community reporting/assistance only and are best effort. 
```

> If Yes: Then a support ticket is the best way for us to assist you with the provider.  Repo issues are for community reporting/assistance only and are best effort. 

Reviewer: ```suggestion
If Yes: Then a support ticket is the best way for us to assist you with the Terraform MongoDB Atlas Provider.  Repo issues are for community reporting/assistance only and are best effort. 
```

Author: Done!

> 
> Do you have a MongoDB Atlas Support contract?  
> 
> If Yes: Then a support ticket is the best way for us to assist you with the provider.  Repo issues are for community reporting/assistance only and are best effort. 
> --->

Reviewer 2: ```suggestion

Do you have a MongoDB Atlas Support contract?  

If Yes: Then a support ticket is the best way for us to assist you with the provider.  Repo issues are for community reporting/assistance only and are best effort. 
```

Author: I didn't add this one as there were no changes that I saw (was it to remove the arrow perhaps?)

Reviewer 2: > was it to remove the arrow perhaps?

yes, there is already an arrow below

Author: Ah, got it!  Fixed.

> If Yes: Then a [support ticket](https://support.mongodb.com/) is the best way for us to assist you with the Terraform MongoDB Atlas Provider.  Repo issues are for community reporting/assistance only and are best effort. 

Reviewer 3: This is clear but I think it could be more concise! Suggest something like the following:

If you have an active MongoDB Atlas Support contract, the best way for us to assist you with the Terraform MongoDB Atlas Provider is through a [support ticket](https://support.mongodb.com/).

Repository issues are only for community reporting and assistance.

Author: Thank you @zach-carr - changed to this.   Much nicer!
